server/src/main/java/org/apache/iotdb/db/query/fill/PreviousFill.java

```diff
@@ -141,15 +141,13 @@ private TimeValuePair retrieveValidLastPointFromSeqFiles() throws IOException {
           FileLoaderUtils.loadTimeSeriesMetadata(
               resource, seriesPath, context, timeFilter, allSensors);
       if (timeseriesMetadata != null) {
-        if (timeseriesMetadata.getStatistics().canUseStatistics()
-            && endtimeContainedByTimeFilter(timeseriesMetadata.getStatistics())) {
+        if (endtimeContainedByTimeFilter(timeseriesMetadata.getStatistics())) {
```

Comment:
Why you delete the judgement `timeseriesMetadata.getStatistics().canUseStatistics()`.
What if the file has been modified and timeseriesMetadata can not been used, for example, it has been deleted from the endTime of it.

Reply:
Fixed